web/pdf_find_bar.js

```diff
@@ -80,6 +81,10 @@ var PDFFindBar = (function PDFFindBarClosure() {
     this.caseSensitive.addEventListener('click', function() {
       self.dispatchEvent('casesensitivitychange');
     });
+
+    this.phraseSearch.addEventListener('click', function() {
```

Comment:
Maybe consider putting a null check around this. Otherwise, adding this line will break customized viewers that have phraseSearch set to null. (In fact, perhaps all of these event viewers should add null checks)